src/Util/Frame.php

```diff
@@ -49,7 +60,7 @@ public function getFile()
     }
 
     /**
-     * @return int|null
+     * @return integer|null
```

Comment:
It is not. The point is here that it is documentation. So if your argument is that `integer` is invalid because`int` is the type (which is false actually) then `MyClass[]` should be invalid as well. The fact that `MyClass[]` is a common pattern shows, that the dockblocks do not have to be strongly tied to the code internals, but should make sense and give the developer a perfect feedback about the code. Using integer perfectly meets this expectation.

And again, why does "others use it" matter? I hate this everyone-should-follow-the-same-style mentality, especially if I proven that I follow another logic, I find other things important (which is (again) that the type name is integer, not int).